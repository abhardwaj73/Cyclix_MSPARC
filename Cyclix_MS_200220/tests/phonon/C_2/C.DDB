***************************************************************************
                            Phononic properties                            
***************************************************************************

Dynamical matrix before symmetrization (not shifted) (Ha/Bohr^2) for q = [      0.0000000000       0.0000000000       0.0000000000]:
     -0.2328452878 +       0.0000000000i	     -0.0010296351 +       0.0000000000i	      0.0376818098 +       0.0000000000i	      0.2990652753 +       0.0000000000i	      0.0001984169 +       0.0000000000i	      0.0174182078 +       0.0000000000i	      0.1276848841 +       0.0000000000i	      0.0252667852 +       0.0000000000i	      0.0062526138 +       0.0000000000i	     -0.1612217230 +       0.0000000000i	     -0.0314107978 +       0.0000000000i	     -0.0614627280 +       0.0000000000i	
     -0.0010296351 +       0.0000000000i	      0.6273212627 +       0.0000000000i	     -0.0046723222 +       0.0000000000i	      0.0005388125 +       0.0000000000i	     -0.2146818508 +       0.0000000000i	      0.0014337279 +       0.0000000000i	     -0.0369732888 +       0.0000000000i	      0.1878604862 +       0.0000000000i	     -0.0005842965 +       0.0000000000i	      0.1181951995 +       0.0000000000i	     -0.5964735141 +       0.0000000000i	      0.0038350926 +       0.0000000000i	
      0.0376818098 +       0.0000000000i	     -0.0046723222 +       0.0000000000i	      0.6849740178 +       0.0000000000i	     -0.0172089127 +       0.0000000000i	      0.0012531014 +       0.0000000000i	     -0.6788673839 +       0.0000000000i	      0.0063555800 +       0.0000000000i	      0.0008439529 +       0.0000000000i	      0.2351591688 +       0.0000000000i	      0.0601713342 +       0.0000000000i	      0.0159095268 +       0.0000000000i	     -0.2395987217 +       0.0000000000i	
      0.2990652753 +       0.0000000000i	      0.0005388125 +       0.0000000000i	     -0.0172089127 +       0.0000000000i	     -0.2330930814 +       0.0000000000i	      0.0001972608 +       0.0000000000i	     -0.0379879193 +       0.0000000000i	     -0.1606507179 +       0.0000000000i	     -0.0320938489 +       0.0000000000i	      0.0614253468 +       0.0000000000i	      0.1277392877 +       0.0000000000i	      0.0250006628 +       0.0000000000i	     -0.0061048241 +       0.0000000000i	
      0.0001984169 +       0.0000000000i	     -0.2146818508 +       0.0000000000i	      0.0012531014 +       0.0000000000i	      0.0001972608 +       0.0000000000i	      0.6329124553 +       0.0000000000i	     -0.0012427559 +       0.0000000000i	      0.1198334683 +       0.0000000000i	     -0.6019094503 +       0.0000000000i	      0.0003609806 +       0.0000000000i	     -0.0372520676 +       0.0000000000i	      0.1878164731 +       0.0000000000i	     -0.0003649381 +       0.0000000000i	
      0.0174182078 +       0.0000000000i	      0.0014337279 +       0.0000000000i	     -0.6788673839 +       0.0000000000i	     -0.0379879193 +       0.0000000000i	     -0.0012427559 +       0.0000000000i	      0.6854963655 +       0.0000000000i	     -0.0609626413 +       0.0000000000i	     -0.0117212369 +       0.0000000000i	     -0.2400863219 +       0.0000000000i	     -0.0063110818 +       0.0000000000i	     -0.0018501905 +       0.0000000000i	      0.2351565711 +       0.0000000000i	
      0.1276848841 +       0.0000000000i	     -0.0369732888 +       0.0000000000i	      0.0063555800 +       0.0000000000i	     -0.1606507179 +       0.0000000000i	      0.1198334683 +       0.0000000000i	     -0.0609626413 +       0.0000000000i	     -0.1963050597 +       0.0000000000i	     -0.1651803149 +       0.0000000000i	      0.0377580150 +       0.0000000000i	      0.2747621497 +       0.0000000000i	      0.0972468694 +       0.0000000000i	      0.0167343747 +       0.0000000000i	
      0.0252667852 +       0.0000000000i	      0.1878604862 +       0.0000000000i	      0.0008439529 +       0.0000000000i	     -0.0320938489 +       0.0000000000i	     -0.6019094503 +       0.0000000000i	     -0.0117212369 +       0.0000000000i	     -0.1651803149 +       0.0000000000i	      0.6006236817 +       0.0000000000i	      0.0071390962 +       0.0000000000i	      0.0975056609 +       0.0000000000i	     -0.1967842319 +       0.0000000000i	      0.0037153333 +       0.0000000000i	
      0.0062526138 +       0.0000000000i	     -0.0005842965 +       0.0000000000i	      0.2351591688 +       0.0000000000i	      0.0614253468 +       0.0000000000i	      0.0003609806 +       0.0000000000i	     -0.2400863219 +       0.0000000000i	      0.0377580150 +       0.0000000000i	      0.0071390962 +       0.0000000000i	      1.5667885815 +       0.0000000000i	     -0.0171250497 +       0.0000000000i	     -0.0028021983 +       0.0000000000i	     -1.5620556261 +       0.0000000000i	
     -0.1612217230 +       0.0000000000i	      0.1181951995 +       0.0000000000i	      0.0601713342 +       0.0000000000i	      0.1277392877 +       0.0000000000i	     -0.0372520676 +       0.0000000000i	     -0.0063110818 +       0.0000000000i	      0.2747621497 +       0.0000000000i	      0.0975056609 +       0.0000000000i	     -0.0171250497 +       0.0000000000i	     -0.1965871561 +       0.0000000000i	     -0.1633483982 +       0.0000000000i	     -0.0366062510 +       0.0000000000i	
     -0.0314107978 +       0.0000000000i	     -0.5964735141 +       0.0000000000i	      0.0159095268 +       0.0000000000i	      0.0250006628 +       0.0000000000i	      0.1878164731 +       0.0000000000i	     -0.0018501905 +       0.0000000000i	      0.0972468694 +       0.0000000000i	     -0.1967842319 +       0.0000000000i	     -0.0028021983 +       0.0000000000i	     -0.1633483982 +       0.0000000000i	      0.5954979169 +       0.0000000000i	     -0.0112343351 +       0.0000000000i	
     -0.0614627280 +       0.0000000000i	      0.0038350926 +       0.0000000000i	     -0.2395987217 +       0.0000000000i	     -0.0061048241 +       0.0000000000i	     -0.0003649381 +       0.0000000000i	      0.2351565711 +       0.0000000000i	      0.0167343747 +       0.0000000000i	      0.0037153333 +       0.0000000000i	     -1.5620556261 +       0.0000000000i	     -0.0366062510 +       0.0000000000i	     -0.0112343351 +       0.0000000000i	      1.5663029278 +       0.0000000000i	

Dynamical matrix after symmetrization (not shifted) (Ha/Bohr^2) for q = [      0.0000000000       0.0000000000       0.0000000000]:
     -0.2328452878 +       0.0000000000i	     -0.0010296351 +       0.0000000000i	      0.0376818098 +       0.0000000000i	      0.2990652753 +       0.0000000000i	      0.0001984169 +       0.0000000000i	      0.0174182078 +       0.0000000000i	      0.1276848841 +       0.0000000000i	      0.0252667852 +       0.0000000000i	      0.0062526138 +       0.0000000000i	     -0.1612217230 +       0.0000000000i	     -0.0314107978 +       0.0000000000i	     -0.0614627280 +       0.0000000000i	
     -0.0010296351 +       0.0000000000i	      0.6273212627 +       0.0000000000i	     -0.0046723222 +       0.0000000000i	      0.0005388125 +       0.0000000000i	     -0.2146818508 +       0.0000000000i	      0.0014337279 +       0.0000000000i	     -0.0369732888 +       0.0000000000i	      0.1878604862 +       0.0000000000i	     -0.0005842965 +       0.0000000000i	      0.1181951995 +       0.0000000000i	     -0.5964735141 +       0.0000000000i	      0.0038350926 +       0.0000000000i	
      0.0376818098 +       0.0000000000i	     -0.0046723222 +       0.0000000000i	      0.6849740178 +       0.0000000000i	     -0.0172089127 +       0.0000000000i	      0.0012531014 +       0.0000000000i	     -0.6788673839 +       0.0000000000i	      0.0063555800 +       0.0000000000i	      0.0008439529 +       0.0000000000i	      0.2351591688 +       0.0000000000i	      0.0601713342 +       0.0000000000i	      0.0159095268 +       0.0000000000i	     -0.2395987217 +       0.0000000000i	
      0.2990652753 +       0.0000000000i	      0.0005388125 +       0.0000000000i	     -0.0172089127 +       0.0000000000i	     -0.2330930814 +       0.0000000000i	      0.0001972608 +       0.0000000000i	     -0.0379879193 +       0.0000000000i	     -0.1606507179 +       0.0000000000i	     -0.0320938489 +       0.0000000000i	      0.0614253468 +       0.0000000000i	      0.1277392877 +       0.0000000000i	      0.0250006628 +       0.0000000000i	     -0.0061048241 +       0.0000000000i	
      0.0001984169 +       0.0000000000i	     -0.2146818508 +       0.0000000000i	      0.0012531014 +       0.0000000000i	      0.0001972608 +       0.0000000000i	      0.6329124553 +       0.0000000000i	     -0.0012427559 +       0.0000000000i	      0.1198334683 +       0.0000000000i	     -0.6019094503 +       0.0000000000i	      0.0003609806 +       0.0000000000i	     -0.0372520676 +       0.0000000000i	      0.1878164731 +       0.0000000000i	     -0.0003649381 +       0.0000000000i	
      0.0174182078 +       0.0000000000i	      0.0014337279 +       0.0000000000i	     -0.6788673839 +       0.0000000000i	     -0.0379879193 +       0.0000000000i	     -0.0012427559 +       0.0000000000i	      0.6854963655 +       0.0000000000i	     -0.0609626413 +       0.0000000000i	     -0.0117212369 +       0.0000000000i	     -0.2400863219 +       0.0000000000i	     -0.0063110818 +       0.0000000000i	     -0.0018501905 +       0.0000000000i	      0.2351565711 +       0.0000000000i	
      0.1276848841 +       0.0000000000i	     -0.0369732888 +       0.0000000000i	      0.0063555800 +       0.0000000000i	     -0.1606507179 +       0.0000000000i	      0.1198334683 +       0.0000000000i	     -0.0609626413 +       0.0000000000i	     -0.1963050597 +       0.0000000000i	     -0.1651803149 +       0.0000000000i	      0.0377580150 +       0.0000000000i	      0.2747621497 +       0.0000000000i	      0.0972468694 +       0.0000000000i	      0.0167343747 +       0.0000000000i	
      0.0252667852 +       0.0000000000i	      0.1878604862 +       0.0000000000i	      0.0008439529 +       0.0000000000i	     -0.0320938489 +       0.0000000000i	     -0.6019094503 +       0.0000000000i	     -0.0117212369 +       0.0000000000i	     -0.1651803149 +       0.0000000000i	      0.6006236817 +       0.0000000000i	      0.0071390962 +       0.0000000000i	      0.0975056609 +       0.0000000000i	     -0.1967842319 +       0.0000000000i	      0.0037153333 +       0.0000000000i	
      0.0062526138 +       0.0000000000i	     -0.0005842965 +       0.0000000000i	      0.2351591688 +       0.0000000000i	      0.0614253468 +       0.0000000000i	      0.0003609806 +       0.0000000000i	     -0.2400863219 +       0.0000000000i	      0.0377580150 +       0.0000000000i	      0.0071390962 +       0.0000000000i	      1.5667885815 +       0.0000000000i	     -0.0171250497 +       0.0000000000i	     -0.0028021983 +       0.0000000000i	     -1.5620556261 +       0.0000000000i	
     -0.1612217230 +       0.0000000000i	      0.1181951995 +       0.0000000000i	      0.0601713342 +       0.0000000000i	      0.1277392877 +       0.0000000000i	     -0.0372520676 +       0.0000000000i	     -0.0063110818 +       0.0000000000i	      0.2747621497 +       0.0000000000i	      0.0975056609 +       0.0000000000i	     -0.0171250497 +       0.0000000000i	     -0.1965871561 +       0.0000000000i	     -0.1633483982 +       0.0000000000i	     -0.0366062510 +       0.0000000000i	
     -0.0314107978 +       0.0000000000i	     -0.5964735141 +       0.0000000000i	      0.0159095268 +       0.0000000000i	      0.0250006628 +       0.0000000000i	      0.1878164731 +       0.0000000000i	     -0.0018501905 +       0.0000000000i	      0.0972468694 +       0.0000000000i	     -0.1967842319 +       0.0000000000i	     -0.0028021983 +       0.0000000000i	     -0.1633483982 +       0.0000000000i	      0.5954979169 +       0.0000000000i	     -0.0112343351 +       0.0000000000i	
     -0.0614627280 +       0.0000000000i	      0.0038350926 +       0.0000000000i	     -0.2395987217 +       0.0000000000i	     -0.0061048241 +       0.0000000000i	     -0.0003649381 +       0.0000000000i	      0.2351565711 +       0.0000000000i	      0.0167343747 +       0.0000000000i	      0.0037153333 +       0.0000000000i	     -1.5620556261 +       0.0000000000i	     -0.0366062510 +       0.0000000000i	     -0.0112343351 +       0.0000000000i	      1.5663029278 +       0.0000000000i	

Phonon_frequencies (cm-1) for q = [      0.0000000000       0.0000000000       0.0000000000]:
      0.0000000000 +    1356.4631371453i	      0.0000000000 +     716.5368347283i	      0.0000000000 +      98.6018902821i	     39.4684111201 +       0.0000000000i	    232.7629271936 +       0.0000000000i	    248.7388565731 +       0.0000000000i	    309.0876842572 +       0.0000000000i	    463.9233437705 +       0.0000000000i	   1354.9046234804 +       0.0000000000i	   1656.7038802543 +       0.0000000000i	   1904.2787233869 +       0.0000000000i	   2674.1550701281 +       0.0000000000i	
 Phonon modes for q = [      0.0000000000       0.0000000000       0.0000000000]:
     -0.4994162838	      0.4961298425	     -0.0002679515	     -0.0029772264	     -0.0771608956	     -0.4863810371	     -0.0021191004	     -0.5115277261	     -0.0005087874	      0.0007851520	      0.0004509428	     -0.0155922516	
     -0.0004413239	      0.0008682522	      0.4995617911	      0.0002367040	     -0.0029572377	      0.0006010181	     -0.5004785264	      0.0019562072	      0.5037034042	      0.0020213264	     -0.4962049821	      0.0021757931	
      0.0647853832	      0.0026200445	     -0.0004598424	      0.4803114551	     -0.5139787995	      0.0450250906	     -0.0010171706	     -0.0224535988	      0.0012526702	      0.6844046954	      0.0070692383	     -0.1729991038	
      0.4993351077	     -0.4971334305	      0.0006243138	      0.0025769825	      0.0488353313	     -0.4901061855	     -0.0028068822	     -0.5105631703	      0.0006222146	      0.0010544583	     -0.0001830403	     -0.0155771401	
     -0.0000700140	      0.0000909900	      0.4999273665	      0.0003291442	     -0.0014721052	     -0.0005820638	      0.5001362240	     -0.0014376118	      0.4967613654	     -0.0065868559	      0.5031060380	     -0.0008747732	
      0.0648225049	      0.0023864658	     -0.0004940405	      0.4802886033	     -0.5153595239	     -0.0156211181	      0.0003988075	      0.0224949865	     -0.0053547552	     -0.6846473977	     -0.0050483284	      0.1731274134	
      0.4826304307	      0.4941283268	     -0.0984171422	      0.0020840423	      0.0507406996	     -0.5020275433	     -0.0961071371	      0.4784726630	      0.0975348005	      0.0343324066	      0.0983770378	     -0.0081271175	
      0.0960669892	      0.0982945465	      0.4906578869	      0.0008345018	      0.0106910787	     -0.1008972161	      0.4901679041	      0.0936707830	     -0.4865815906	      0.0145301278	     -0.4940209411	     -0.0008129910	
     -0.0649572026	     -0.0020695970	     -0.0002700213	      0.5189399298	      0.4759052492	     -0.0045225993	      0.0004372494	      0.0115238396	      0.0021798193	     -0.1731730654	     -0.0047056858	     -0.6854359919	
     -0.4827818413	     -0.4933097890	     -0.0977111728	     -0.0026771881	     -0.0804635157	     -0.4984209600	      0.0982102242	      0.4787079715	      0.0981235814	      0.0365424723	     -0.0963777814	     -0.0074724950	
     -0.0957766300	     -0.0975344918	      0.4904396186	     -0.0000389024	     -0.0126251162	     -0.0987507505	     -0.4901273614	      0.0975222679	     -0.4935268570	      0.0042356415	      0.4873771389	     -0.0036244757	
     -0.0649354813	     -0.0019442809	     -0.0002578145	      0.5189385660	      0.4754705608	     -0.0234502696	      0.0001155972	     -0.0115556981	      0.0019258355	      0.1733981277	      0.0026843995	      0.6853092432	
