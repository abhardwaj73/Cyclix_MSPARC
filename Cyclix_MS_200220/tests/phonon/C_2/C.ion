#=========================
# format of ion file
#=========================
# ATOM_TYPE: <atom type name> <valence charge>
# N_TYPE_ATOM: <num of atoms of this type>
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX:
# <xrelax> <yrelax> <zrelax>
# ...


# Reminder: when changing number of atoms, change the RELAX flags accordingly
#           as well.

ATOM_TYPE: C              # atom type followed with valence charge
N_TYPE_ATOM: 4               # number of atoms of this type
PSEUDO_POT: C_local.psp8
COORD_FRAC:
0.588157665635968                   0.001             0.001
0.588157665635968   0.000000000000000   0.333333333333333
0.588157665635968   0.500000000000000   0.500000000000000
0.588157665635968   0.500000000000000   0.833333333333333


