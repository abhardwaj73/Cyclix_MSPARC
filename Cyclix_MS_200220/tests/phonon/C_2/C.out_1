***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 22-Jan-2022  Start time: 16:17:02                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.785398 8.001000 
FD_GRID: 50 24 21
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 8 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 10
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/C_2/C.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.400000 (Bohr)
Mesh spacing in y-direction        :  0.384948 (Bohr)
Mesh spacing in z direction        :  0.381000 (Bohr)
Number of symmetry adapted k-points:  5
Output printed to                  :  ../tests/phonon/C_2/C.out_1
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.32 0.63 7.24
Number of atoms of type 1          :  2
Estimated total memory usage       :  150.22 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.3888945925E+00        1.880E-01        13.770
2            -5.4347216982E+00        4.764E-02        3.710
3            -5.4325338106E+00        7.182E-02        3.705
4            -5.4399444951E+00        4.664E-02        3.555
5            -5.4433520795E+00        1.945E-02        3.463
6            -5.4434383498E+00        6.339E-03        3.507
7            -5.4434657066E+00        3.673E-03        3.491
8            -5.4434730701E+00        3.181E-03        3.628
9            -5.4434772103E+00        8.077E-04        3.445
10           -5.4434780380E+00        2.860E-04        3.692
11           -5.4434778851E+00        1.282E-04        3.305
12           -5.4434778728E+00        4.937E-05        3.289
13           -5.4434778659E+00        1.384E-05        3.852
14           -5.4434778763E+00        4.853E-06        4.237
15           -5.4434778706E+00        1.928E-06        3.850
16           -5.4434778698E+00        2.403E-06        3.983
17           -5.4434778709E+00        1.409E-06        3.686
18           -5.4434778716E+00        5.419E-07        3.478
Total number of SCF: 18    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.4434778716E+00 (Ha/atom)
Total free energy                  : -1.0886955743E+01 (Ha)
Band structure energy              : -3.1165404506E+00 (Ha)
Exchange correlation energy        : -2.7796948640E+00 (Ha)
Self and correction energy         : -2.1917339884E+01 (Ha)
Entropy*kb*T                       : -1.0170546520E-03 (Ha)
Fermi level                        : -2.3290760010E-01 (Ha)
Average force                      :  2.0404709551E-02 (Ha/Bohr)
Maximum force                      :  2.0404819051E-02 (Ha/Bohr)
Time for force calculation         :  2.664 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  98.897 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
