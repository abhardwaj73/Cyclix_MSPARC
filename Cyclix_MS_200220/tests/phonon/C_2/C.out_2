***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 22-Jan-2022  Start time: 16:31:37                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.785398 8.001000 
FD_GRID: 50 24 21
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 24
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/C_2/C.out_2
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.400000 (Bohr)
Mesh spacing in y-direction        :  0.384948 (Bohr)
Mesh spacing in z direction        :  0.381000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/phonon/C_2/C.out_2
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.32 0.63 7.24
Number of atoms of type 1          :  8
Estimated total memory usage       :  90.60 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.6835502821E+00        1.694E-01        7.886
2            -5.7046717056E+00        1.223E-01        6.879
3            -5.7090631968E+00        4.380E-02        3.829
4            -5.7098994039E+00        1.193E-02        3.767
5            -5.7100173335E+00        6.550E-03        3.908
6            -5.7100335005E+00        3.456E-03        3.894
7            -5.7100372412E+00        1.667E-03        3.727
8            -5.7100375936E+00        6.592E-04        3.740
9            -5.7100383445E+00        3.154E-04        3.597
10           -5.7100386870E+00        1.832E-04        3.755
11           -5.7100387676E+00        1.442E-04        3.514
12           -5.7100387982E+00        4.978E-05        3.581
13           -5.7100387905E+00        1.110E-05        3.481
14           -5.7100387859E+00        6.565E-06        3.467
15           -5.7100387871E+00        2.938E-06        3.454
16           -5.7100387916E+00        1.492E-06        3.462
17           -5.7100387909E+00        5.854E-07        3.355
Total number of SCF: 17    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.7100387909E+00 (Ha/atom)
Total free energy                  : -4.5680310327E+01 (Ha)
Band structure energy              : -1.5991890709E+01 (Ha)
Exchange correlation energy        : -1.3989945184E+01 (Ha)
Self and correction energy         : -8.7621264087E+01 (Ha)
Entropy*kb*T                       : -6.5967658348E-10 (Ha)
Fermi level                        : -2.3829889403E-01 (Ha)
Average force                      :  3.1515160426E-02 (Ha/Bohr)
Maximum force                      :  3.5795232763E-02 (Ha/Bohr)
Time for force calculation         :  2.575 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  75.014 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
