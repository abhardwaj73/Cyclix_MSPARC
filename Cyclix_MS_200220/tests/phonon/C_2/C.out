***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 22-Jan-2022  Start time: 16:11:45                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.785398 8.001000 
FD_GRID: 50 24 21
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 10
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/C_2/C.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.400000 (Bohr)
Mesh spacing in y-direction        :  0.384948 (Bohr)
Mesh spacing in z direction        :  0.381000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/phonon/C_2/C.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.32 0.63 7.24
Number of atoms of type 1          :  2
Estimated total memory usage       :  79.62 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.3933184060E+00        2.368E-01        4.527
2            -5.3885159675E+00        1.905E-01        2.113
3            -5.4574304119E+00        8.851E-02        1.716
4            -5.4696383118E+00        4.920E-02        1.786
5            -5.4732260284E+00        2.484E-02        1.962
6            -5.4740513665E+00        1.129E-02        1.806
7            -5.4741061274E+00        8.513E-03        1.757
8            -5.4741016234E+00        3.032E-03        1.797
9            -5.4741157870E+00        3.449E-03        1.710
10           -5.4741249437E+00        7.139E-04        1.769
11           -5.4741255283E+00        4.371E-04        1.661
12           -5.4741256454E+00        2.094E-04        1.659
13           -5.4741252758E+00        5.307E-04        1.632
14           -5.4741256921E+00        6.486E-05        1.641
15           -5.4741256073E+00        1.008E-05        1.724
16           -5.4741256076E+00        6.039E-06        1.474
17           -5.4741256070E+00        1.638E-06        1.567
18           -5.4741256044E+00        6.534E-07        1.429
Total number of SCF: 18    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.4741256044E+00 (Ha/atom)
Total free energy                  : -1.0948251209E+01 (Ha)
Band structure energy              : -3.2289462712E+00 (Ha)
Exchange correlation energy        : -2.7788771827E+00 (Ha)
Self and correction energy         : -2.1917339884E+01 (Ha)
Entropy*kb*T                       : -1.5260238305E-03 (Ha)
Fermi level                        : -2.5545245508E-01 (Ha)
Average force                      :  2.4300115324E-02 (Ha/Bohr)
Maximum force                      :  2.4300411246E-02 (Ha/Bohr)
Time for force calculation         :  0.777 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  37.535 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
